src/Microsoft.AspNetCore.Sockets.Client.Http/WebSocketsTransport.Log.cs

```diff
@@ -65,7 +65,7 @@ private static class Log
             private static readonly Action<ILogger, Exception> _cancelMessage =
                 LoggerMessage.Define(LogLevel.Debug, new EventId(18, "CancelMessage"), "Canceled passing message to application.");
 
-            public static void StartTransport(ILogger logger, TransferMode transferMode)
+            public static void StartTransport(ILogger logger, TransferFormat transferMode)
```

Comment:
transferFormat parameter name. Maybe do a search and find and replace to be sure